templates/roles_README.md

```diff
@@ -0,0 +1,21 @@
+# Role Name
+
+A brief summary of a role.
+
+### Usage
+
+```
+Bash command to execute a playbook
```

Comment:
roles are not really executed as standalone, I would probably skip this section for roles .

Reply:
I added it here because of: https://github.com/kubevirt/kubevirt-ansible/tree/master/roles/cdi#usage